### PR TITLE
New landing page, a scroll indicator

### PR DESCRIPTION
**Pull request by WebDevWorker**
ScrollIndicator

### CODE REVIEW

File: src/components/ScrollIndicator/ScrollIndicator.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { createBemBlockBuilder } from '../../utils';
```
Comment (ViktorSoroka07) [inline src/components/ScrollIndicator/ScrollIndicator.jsx]:
```suggestion
import React from 'react';

import { createBemBlockBuilder } from '../../utils';
```

File: src/components/InstallationPage/InstallationPage.scss [hunk 2]
```diff
@@ -12,6 +12,25 @@
...
+    }
+
+    &-indicator {
+      flex: 3;
+    }
+
+    &-content {
+      
+      flex: 7;
+    }
```
Comment (ViktorSoroka07) [inline src/components/InstallationPage/InstallationPage.scss]:
```suggestion
    &-content {
      flex: 7;
    }
```

File: src/components/InstallationPage/InstallationPage.jsx [hunk 3]
```diff
@@ -52,9 +78,51 @@ export const InstallationPage = () => {
...
+            <div
+              key="section-4"
+              name="section-4"
+              id="part-4"
+              style={{
+                width: '100%',
+                height: '100vh',
+                textAlign: 'center',
+                background: '#F4EAD5',
+              }}
```
Comment (ViktorSoroka07) [inline src/components/InstallationPage/InstallationPage.jsx]: Why not CSS? Also, this way we can use CSS variables

File: src/components/ScrollIndicator/ScrollIndicator.jsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+                  className={getBlocksWith('__link')}
+                  activeClass="active"
+                  to={section.id}
+                  spy={true}
+                  smooth={true}
+                  offset={-70}
+                  duration={500}
+                >
+                  <span>{getSubstring(section.title, 0)}</span>
+                  <br />
```
Comment (ViktorSoroka07) [inline src/components/ScrollIndicator/ScrollIndicator.jsx]: Should we use `br`? I remember some discussion was about not using it

File: src/components/ScrollIndicator/ScrollIndicator.jsx [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+import '../InstallationPage/InstallationPage.scss';
+import './ScrollIndicator.scss';
+
+const getBlocksWith = createBemBlockBuilder(['indicatory']);
+
+import { Link } from 'react-scroll';
+
+export const ScrollIndicator = ({ sections }) => {
+  const getSubstring = (title, substringNumber) => title.split('\n')[substringNumber];
```
Comment (ViktorSoroka07) [inline src/components/ScrollIndicator/ScrollIndicator.jsx]: Why do we need it? Maybe it is better to discuss with designer. I am not sure this is good pattern

File: src/components/InstallationPage/InstallationPage.jsx (hunk 3 above)
Comment (WebDevWorker) [reply src/components/InstallationPage/InstallationPage.jsx]: I'll remove it after merging it with my last PR. I created this block for the last section that is not existing at the moment but still can test our scroll indicator. So? we don't need the styles in a scss file.

File: src/components/ScrollIndicator/ScrollIndicator.jsx (hunk 4 above)
Comment (WebDevWorker) [reply src/components/ScrollIndicator/ScrollIndicator.jsx:31]: Yes, you are right, but that discussion was in another context, where I tried to separate mock text data that should be copyable

File: src/components/ScrollIndicator/ScrollIndicator.jsx (hunk 5 above)
Comment (WebDevWorker) [reply src/components/ScrollIndicator/ScrollIndicator.jsx:12]: What is the bad point here? I could try to reorganize it

**Review by ViktorSoroka07 — APPROVED** [review]

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**